sdks/java/portability/src/main/java/org/apache/beam/sdks/java/portability/PortableRunner.java

```diff
@@ -15,7 +15,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.beam.runners.reference;
+package org.apache.beam.sdks.java.portability;
```

Comment:
```suggestion
package org.apache.beam.runners.portability;
```

Why not move this here?